backend/src/gpml/handler/review.clj

```diff
@@ -1,13 +1,86 @@
 (ns gpml.handler.review
   (:require
+   [clojure.java.jdbc :as jdbc]
+   [gpml.constants :as constants]
    [gpml.db.stakeholder :as db.stakeholder]
+   [gpml.db.review :as db.review]
    [integrant.core :as ig]
    [ring.util.response :as resp]))
 
 (defn get-reviewers [db]
   (let [conn (:spec db)]
     (resp/response (db.stakeholder/get-reviewers conn))))
 
-(defmethod ig/init-key :gpml.handler.review/get-reviewers [_ {:keys [db]}]
+(defn get-review [db topic-type topic-id]
+  (let [conn (:spec db)
+        topic-name (cond
+                     (contains? (set constants/resource-types) topic-type) "resource"
```

Comment:
perhaps it's time to use `set` in `gpml.constants` ... I found in the code lots of `(set constants/resource-types)`